app/src/main/java/ch/epfl/skysync/navigation/Route.kt

```diff
@@ -0,0 +1,12 @@
+package ch.epfl.skysync.navigation
+
+/** Define the paths used for the navigation in the app */
+object Route {
+  const val MAIN = "Main graph"
```

Comment:
What's the difference between "Main" Route and "Home" Route?

Reply:
See [here](https://github.com/SkySync-EPFL/skysync/pull/71#discussion_r1541202006) (I had the same question)